spec/views/user/user_show_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.feature 'User show page', type: :feature do
+  include Rails.application.routes.url_helpers
+  let!(:user) { User.create(name: 'Menen', photo: 'menen.jpg', postcounter: 5) }
+
+  before do
+    visit user_path(user)
+  end
+
+  it 'show user profile picture' do
+    expect(page).to have_css("img[src*='menen.jpg']")
+  end
+  it 'show the users username' do
+    expect(page).to have_content(user.name)
+  end
+
+  it 'showthe number of posts the user has written.' do
+    expect(page).to have_content("Number of posts : #{user.postcounter}")
+  end
+
+  it 'show the bio of the user' do
+    expect(page).to have_content(user.bio)
+  end
+
+  it 'shows the user first three posts' do
+    user.posts.limit(3).each do |post|
+      expect(page).to have_content(post.title)
+    end
+  end
```

Comment:
- You have done well so far, remember you have a method, `recent_posts`, in your user model that gives the first three posts for a user. I suggest you test this method to check for a user's first three posts. Please fix this 🙏